idd/Energy+.idd.in

```diff
@@ -53562,12 +53574,16 @@ Coil:Cooling:DX:VariableSpeed,
         \note curve = a + b*wb + c*wb**2 + d*odb + e*odb**2 + f*wb*odb
         \note wb = entering wet-bulb temperature (C)
         \note odb = air entering temperature seen by the condenser (C)
-   A49; \field Speed 10 Energy Input Ratio Function of Air Flow Fraction Curve Name
+   A49, \field Speed 10 Energy Input Ratio Function of Air Flow Fraction Curve Name
         \type object-list
         \object-list UnivariateFunctions
         \note quadratic curve = a + b*ffa + c*ffa**2
         \note cubic curve = a + b*ffa + c*ffa**2 + d*ffa**3
         \note ffa = Fraction of the full load Air Flow
+   A50; \field Outdoor Temperature Dependent Crankcase Heater Capacity Curve Name
```

Comment:
Hmm, the group of speed fields is currently not extensible, so technically, this is ok to add at the end of the object, but it just seems lost here. @Myoldmopar do we care?

Reply:
Yeah, I'm not certain whether I should put it at the end or near the other crankcase heater fields.